api/swagger/swagger.yaml

```diff
@@ -294,6 +294,23 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /sources:
+    x-swagger-router-controller: sources
+    get:
+      tags:
+        - sources
+      description: Returns list of sources
+      operationId: list
```

Comment:
I'd go with `getSources` here, to comply with the latest Swagger updates (I really need to remove that WIP tag from the PR). This should be reflected in the explorer agent code.